process/block/metablock.go

```diff
@@ -517,7 +515,7 @@ func (mp *metaProcessor) CommitBlock(
 		log.Info(errNotCritical.Error())
 	}
 
-	hdrsToAttestFinality := mp.nextKValidity
+	hdrsToAttestFinality := mp.nextKValidity + 1
```

Comment:
rename variable - You need to roll back to final -1.

Reply:
rename function to removeNotarizedBehindFinalMinusOne

Reply:
done